Xamarin.Forms.Core/Button.cs

```diff
@@ -50,6 +50,30 @@ public class Button : View, IFontElement, ITextElement, IBorderElement, IButtonC
 			propertyChanging: (bindable, oldvalue, newvalue) => ((Button)bindable).OnSourcePropertyChanging((ImageSource)oldvalue, (ImageSource)newvalue),
 			propertyChanged: (bindable, oldvalue, newvalue) => ((Button)bindable).OnSourcePropertyChanged((ImageSource)oldvalue, (ImageSource)newvalue));
 
+		public static readonly BindableProperty PaddingProperty = PaddingElement.PaddingProperty;
+
+		public Thickness Padding
+		{
+			get { return (Thickness)GetValue (PaddingElement.PaddingProperty); }
+			set { SetValue (PaddingElement.PaddingProperty, value); }
+		}
+
+		Thickness IPaddingElement.PaddingDefaultValueCreator ()
+		{
+			return default (Thickness);
+		}
+
+		void IPaddingElement.OnPaddingPropertyChanged (Thickness oldValue, Thickness newValue)
+		{
+			UpdatePadding(newValue);
+			InvalidateMeasureInternal(InvalidationTrigger.MeasureChanged);
+		}
+
+		protected virtual void UpdatePadding (Thickness newValue)
```

Comment:
this isn't necessary. subclass of Button have other ways of detecting Padding changes.